### PR TITLE
test: Add PackageKit to Fedora/RHEL/CentOS images

### PR DESCRIPTION
**Pull request by martinpitt**
We need that to test OS updates (#6724) and soon for AppStream based
service installation.

- [x] rhel-7
- [x] centos-7
- [x] fedora-i386
- [x] fedora-26

### CODE REVIEW

**Comment by martinpitt** [comment]
bot: Image refresh for fedora-i386

**Comment by martinpitt** [comment]
bot: Image refresh for fedora-26

**Comment by martinpitt** [comment]
bot: Image refresh for centos-7

**Comment by martinpitt** [comment]
bot: Image refresh for rhel-7

**Comment by martinpitt** [comment]
The bots don't seem to like me. I'll rebuild/upload the images myself locally.

File: bots/images/scripts/fedora-testing.setup [hunk 1]
```diff
@@ -31,6 +31,7 @@ libssh \
 libvirt \
 libvirt-client \
 NetworkManager-team \
+PackageKit \
```
Comment (petervo) [inline bots/images/scripts/fedora-testing.setup]: I don't see a new fedora-testing image to match this change.

Comment (martinpitt) [inline bots/images/scripts/fedora-testing.setup]: fedora-testing rebuilds have some consistent OpenShift failures (#6745) which need to be investigated first. As we don't run fedora-testing in CI regularly, I thought I could rebuild it separately?

Comment (martinpitt) [inline bots/images/scripts/fedora-testing.setup]: Re-pushed with backing out fedora-testing, and made a note in #6745 about it.

**Pull request merged**